agents/bug_reporter/README.md

```diff
@@ -0,0 +1,23 @@
+# Bug Reporter agent
+
+This Agent files bug reports in GitHub. To use it, need to make the following changes:
+
+1. Create a GitHub access token. To do this, go to your settings page on GitHub, find the
+`Developer settings` section, and click on `Personal access tokens`. Then, click on `Generate new token`.
+Make sure you assign the token `read/write` permissions on Issues.
+
+2. Create a `.env` file in this directory, and add the following environment variable:
+
+```
+GITHUB_ACCESS_TOKEN=<your GitHub access token>
+```
+
+3. Edit the file `main.py` and change the value of the `REPO` variable to match the name of the
```

Comment:
Maybe this should be in the env too?

Reply:
Yeah, I thought of that. The thing is, I considered .env to be mainly used for secrets, not for configuration variables. This was a point I had made a while back that I thought we should allow configuration to be set in an agent.yaml file (and values changed by the user at deployment time) so someone did not need to modify the code for things like this. I could see this getting out of hand if people had a lot of configuration they wanted for their agents, stuffing everything in an env var is kind of unwieldy. I think editing a .env file or the main.py file for this are basically equivalent, so I thought it would be easier to understand the agent code if it was just in the source file.